### DISCUSSION TITLE
Not full declarations for teal-types repo.

### DISCUSSION THREAD

**Discussion by nagolove** [teal-language/tl#644] [General]
Hello! I have some manually written d.tl files for third-party libraries in my project. But this files does not cover full library APIs, for example Lanes. Is it a good point to make some PRs with such partially types?

**Comment by lenscas** [comment]
I would make a PR that adds a "partial" folder that is otherwise the same structure or mark the files as partial in some other way.

Partial .d.tl files are still more useful than no .d.tl file.

**Reply by hishamhm** [reply]
> Partial .d.tl files are still more useful than no .d.tl file.

Agree!

I think a comment inside the .d.tl file stating that the definitions are complete would be enough for now.

I don't think we should add a separate folder because a "complete" definition may become "partial" when a new version is released, etc. There are already incomplete definitions there, it's a best-effort repo, it does not claim to be perfect.
